js/background.js

```diff
@@ -144,3 +144,40 @@ chrome.webRequest.onBeforeRequest.addListener(
     },
     ["blocking"]
 );
+
+chrome.webRequest.onCompleted.addListener(
+    function () {
+      var atb = localStorage['atb'],
+          setATB = localStorage['set_atb'];
+
+      if (atb === undefined) {
+        return;
+      }
+
+      var xhr = new XMLHttpRequest();
+
+      xhr.onreadystatechange = function() {
+        if (xhr.readyState == XMLHttpRequest.DONE) {
+           if (xhr.status == 200) {
+             var curATB = JSON.parse(xhr.responseText);
+             if(curATB.version !== setATB) {
+               localStorage['set_atb'] = curATB.version;
+             }
+           }
+        }
+      };
+
+      xhr.open('GET',
+        'https://duckduckgo.com/collect.js?' + new Date().getTime()
+          + '&type=atb&atb=' + atb + '&set_atb=' + setATB,
```

Comment:
@zachthompson this is a nitpick, but do you mind if I change the timestamp to a random number instead? `Math.ceil(Math.random() * 1e7)`